File: PHP/palindrome.php
<?php
function palindrome($no)
{
    $num = $no;
    $sum = 0;
    while (floor($num)) {
        $rem = $num % 10;
        $sum = $sum * 10 + $rem;
        $num = $num / 10;
    }
    return $sum;
}
$input = 12321;
$noum = palindrome($input);
if ($input == $noum) {
    echo $input . " is a Palindrome number";
} else {
    echo $input . " is not a Palindrome";
}

File: Palindrome.c
#include<stdio.h>
void main()
{
  int n,rem,rev=0;

  printf("Enter a number= ");
  scanf("%d",&n);

  int a=n;
  while(a>0)
  {
    rem=a%10;
    rev=rev*10 + rem; 
    a=a/10;
  }

  if(n==rev)
  {
    printf("%d is a palindrome",n);
  }
  else
  {
     printf("%d is not a palindrome",n);
  }
}
